docs/articles/documentation/recipes/running-tests-with-saucelabs-on-travis-ci/index.md

```diff
@@ -0,0 +1,93 @@
+* [Step 5 - Trigger a Travis CI build](#step-5---trigger-a-travis-ci-build)
+
+The [Example](#example) section describes how to run tests with Travis CI and Sauce Labs for a sample project.
+
+## Step 1 - Enable Travis for your project
+
+1. [Sign in to Travis CI](https://travis-ci.org/auth) with your GitHub account. Travis CI will synchronize your repositories from GitHub. You can see them on your [profile page](https://travis-ci.org/profile).
+2. Еnable Travis CI for a repository you want to build by flicking the switch on.
+
+     ![Enable Travis for a repository](../../../images/travis-step-1-2.png)
+
+3. Add a `.travis.yml` configuration file to the root of your project. This file should contain parameters and commands that instruct Travis CI on how to execute your builds. For more information, see [Customizing the Build](https://docs.travis-ci.com/user/customizing-the-build).
+
+     For Node.js projects, the file can have the following content.
+
+     ```text
+     language: node_js
+     node_js: 6
+     ```
+
+     Commit and push the `.travis.yml` file to your repository.
+
+4. Enable builds on pushes and pull requests in your repository settings. To do this, navigate to the repository, choose **Settings** from the **More Options** menu, and enable the **Build Pushes** and **Build Pull Requests** options in the **General Settings** section.
+
+     ![Enable builds](../../../images/travis-step-1-4.png)
+
+## Step 2 - Configure Travis to use Sauce Labs
+
+1. Save your Sauce Labs username and access key to the SAUCE\_USERNAME and SAUCE\_ACCESS\_KEY environment variables, as described in [SauceLabs documentation](https://wiki.saucelabs.com/display/DOCS/Best+Practice%3A+Use+Environment+Variables+for+Authentication+Credentials).
+2. Go to the repository settings in Travis CI. In the **Environment Variables** section, define the SAUCE\_USERNAME and SAUCE\_ACCESS\_KEY variables (see [Defining Variables in Repository Settings](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings)).
+
+     ![Define variables](../../../images/travis-step-2-2.png)
+
+## Step 3 - Install the Sauce Labs browser provider plugin
+
+To run TestCafe tests in the cloud using Sauce Labs browsers, you need to install the [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs) plugin locally to your project.
+To do this, run the following command.
+
+```bash
+npm install --save-dev testcafe-browser-provider-saucelabs
+```
+
+For more information, see [Browser Provider Plugins](../using-testcafe/common-concepts/browser-support.md#browser-provider-plugins).
+
+## Step 4 - Add the `test` script to package.json
+
+To test a project, Travis runs test scripts. For Node.js projects, the default test script is `npm test`.
+To tell npm on how to run your tests, you need to add the `test` script to the project's package.json file.
```

Comment:
> tell npm on

as Armina once said

> I know why you Russians are always saying 'tell on' and 'inform on' - that's because of 1937

Because both these expressions mean 'to write a report about someone to the authorities'